kujira/assetlist.json

```diff
@@ -2463,6 +2463,33 @@
         }
       ]
     },
+    {
+      "description": "The receipt token of NAMI Protocol",
+      "denom_units": [
+        {
+          "denom": "factory/kujira18thffdpuz8tm3mwwun4uhks8a2v7ye0jmw3fllqhklma82lqkpms20gh9c/unausd",
+          "exponent": 0
+        },
+        {
+          "denom": "nausd",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira18thffdpuz8tm3mwwun4uhks8a2v7ye0jmw3fllqhklma82lqkpms20gh9c/unausd",
+      "name": "NAUSD",
+      "display": "naUSD",
```

Comment:
this display denom is not defined (units are case sensitive). "nausd" is a defined denom, though.